### ISSUE TITLE
Wilson roots correspondence to MW roots, step2a

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/WIL#6]
### Add L-numbers

The main output detailing the root correspondences is [wil_mw.txt](https://github.com/sanskrit-lexicon/WIL/blob/master/wilmwroots/step2a/wil_mw.txt).  In this revision, each correspondence
includes the L-numbers for both Wilson and MW.  For examples:
```
<c>SPa-ROOT</c> <wil>aMSa</wil> <mw>aMS</mw> <wilL>5</wilL> <mwL>9</mwL>
<c>SPa-ROOT</c> <wil>aWa</wil> <mw>aW</mw> <wilL>642</wilL> <mwL>2383,2447.1</mwL>
```
Note there are 2 MW records that correspond in the second case.
See [readme.org](https://github.com/sanskrit-lexicon/WIL/blob/master/wilmwroots/step2a/readme.org) for a brief discussion.

There is still a secondary list of 35 Wilson roots for which there are not (yet) corresponding MW roots.
See [wil_mw_prob.txt](https://github.com/sanskrit-lexicon/WIL/blob/master/wilmwroots/step2a/wil_mw_prob.txt) for this list.

It is likely that most of these can be solved --  anyone want to give it a try?

**Comment by funderburkjim** [comment]
### Why add these L-numbers now?
The reason for adding these L-numbers is so we can somehow use the MW spellings of these roots
when doing dictionary lookups in Wilson.   For instance, we should get  Wilson record L=642  when
we enter the citation 'aW' -- we shouldn't have to use the spelling 'aWa'.    
One way would be to enter corrections to the 'meta' lines of Wilson for these roots. For instance,
here is the information on L=642 in wil.txt:
```
<L>642<pc>015<k1>aWa<k2>aWa
{#aWa#}¦ r. 1st cl. ({#aWati#}) {%With a redundant%} {#i. aWi#}
({#aMWate#}) To go, to go to or towards.

<LEND>
```
All we'd have to do is change the value of '<k1>' in the meta-line, as follows:
```
<L>642<pc>015<k1>aW<k2>aWa
```

Making this change (and similarly for the other roots) to the meta lines in wil.txt, and then
creating wil.xml exactly as we are currently doing would result in displays which would return
the L=642 information when searched for with key 'aW'.

A side effect of this procedure would be that the L=642 information would NOT be found if searching
for 'aWa'.

My thinking at the moment is that this side effect is not a problem.  Does any (hypthetical) reader of this note have an opinion here?

**Comment by gasyoun** [comment]
> A side effect of this procedure would be that the L=642 information would NOT be found if searching
for 'aWa'.

Does not sound good at all.

> My thinking at the moment is that this side effect is not a problem. Does any (hypthetical) reader of this note have an opinion here?

If a word, that is in the book, can't be found as it is, that is not what we want. No meta meta line ghost-word  solution?

**Comment by drdhaval2785** [comment]
I also agree that this is not a good solution. 'aW' and 'aWa' both should access the entry. Not either or.
